app/Repositories/PhotoRepository.php

```diff
@@ -68,19 +62,14 @@ public function create(array $data, $signupId)
         if (isset($data['created_at'])) {
             $photo->created_at = $data['created_at'];
             $photo->updated_at = $data['created_at'];
-            $postEvent->created_at = $data['created_at'];
-            $postEvent->updated_at = $data['created_at'];
 
             $photo->save(['timestamps' => false]);
-            $postEvent->save(['timestamps' => false]);
```

Comment:
This brought up a thought, will there be a way to override the timestamps in the event table if we want to? Currently everything that is being migrated gets backdated. Might be weird to have inconsistencies between signup timestamps and the timestamps of their events.